index.html

```diff
@@ -1,372 +1,1327 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<!-- This site was created in Webflow. https://www.webflow.com -->
+<!-- Last Published: Sun Nov 13 2022 20:58:49 GMT+0000 (Coordinated Universal Time) -->
+<html
+   data-wf-domain="www.sannewijbenga.com"
+   data-wf-page="62d06e0891cd2e8993791ba8"
+   data-wf-site="5c5c83856efacb1a6810474b"
+   lang="en"
+>
+   <head>
+      <meta charset="utf-8" />
+      <title>Decentralized music streaming</title>
 
-<head>
-    <title>Muzikie - Decentralized Music Streaming Platform</title>
+      <meta
+         content="Muzikie is a Decentralized music streaming platform which is fair, transparent, easy to use and democratic. Join music to enjoy music while supporting the future of music."
+         name="description"
+      />
+      <meta
+         content="Muzikie - Decentralized music streaming"
+         property="og:title"
+      />
+      <meta
+         content="Muzikie is a Decentralized music streaming platform which is fair, transparent, easy to use and democratic. Join music to enjoy music while supporting the future of music."
+         property="og:description"
+      />
+      <meta
+         content="Muzikie - Decentralized music streaming"
+         property="twitter:title"
+      />
+      <meta
+         content="Muzikie is a Decentralized music streaming platform which is fair, transparent, easy to use and democratic. Join music to enjoy music while supporting the future of music."
+         property="twitter:description"
+      />
+      <meta property="og:type" content="website" />
+      <meta content="width=device-width, initial-scale=1" name="viewport" />
+      <link href="./dist/style.css" rel="stylesheet" type="text/css" />
 
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="Muzikie - Decentralized Music Streaming Platform">
-    <meta name="keywords" content="music, streaming, blockchain, listen, singer, dj">
+      <script
+         type="text/javascript"
+         src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"
```

Comment:
We have JQuery script pushed to the repo, while here we read from a CDN. Please use one. And if you opt to the local, please use npm.